docs/cli-reference/dfx-canister.md

```diff
@@ -438,6 +438,7 @@ You can use the following optional flags with the `dfx canister install` command
 
 | Flag                  | Description                                                                                                                                                             |
 |-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--argument-file`     | Specifies the file from which to read the argument to pass to the init method.  Stdin may be referred to as `-`.                                                        |
```

Comment:
> Stdin may be referred to as `-`.

This is not tested. Can you maybe add a test for that too?

Reply:
Sure thing.  Done.